tldextract/tldextract.py

```diff
@@ -334,7 +331,7 @@ def get_tlds_from_raw_suffix_list_data(suffix_list_source, include_psl_private_d
         text, _, _ = suffix_list_source.partition('// ===BEGIN PRIVATE DOMAINS===')
 
     tld_iter = (m.group('suffix') for m in PUBLIC_SUFFIX_RE.finditer(text))
-    return frozenset(tld_iter)
+    return list(tld_iter)
```

Comment:
Other code paths besides JSON serializing use this, and will get the wrong type. Try coercing to a `list` only when serializing, e.g.

```python
json.dump(list(my_frozenset_of_tlds), some_file)
```